extensions/producthunt/src/components/ProductDetailView.tsx

```diff
@@ -146,7 +147,7 @@ export function ProductDetailView({
                       });
                       open(maker.profileUrl);
                     } else if (maker.username) {
-                      const profileUrl = `https://www.producthunt.com/@${maker.username}`;
+                      const profileUrl = `${HOST_URL}/@${maker.username}`;
```

Comment:
As you already have the slash in the constant, you could remove it here ;) 
```suggestion
                      const profileUrl = `${HOST_URL}@${maker.username}`;
```